src/pile.js

```diff
@@ -1263,25 +1292,25 @@ const createPile = (
       labelGraphics.removeChildren();
     }
 
-    const {
-      pileLabelHeight,
-      pileLabelAlign,
-      pileLabelFontSize,
-      pileLabelStackAlign,
-      pileLabelText,
-      piles
-    } = store.state;
+    const { pileLabelText, piles } = store.state;
 
-    const bounds = getContentBounds();
+    const pileLabelAlign = getPileProp(store.state.pileLabelAlign, piles[id]);
```

Comment:
Let's keep the previous way of destructing the state instead of repeatedly getting the state with `store.state`. To avoid name clashes you can just remove the `pile` prefix from `const pileLabelAlign` and the other variables :)